src/wanna/cli/plugins/managed_notebook_plugin.py

```diff
@@ -111,3 +120,28 @@ def build(
         workdir = pathlib.Path(file).parent.resolve()
         nb_service = ManagedNotebookService(config=config, workdir=workdir)
         nb_service.build()
+
+    @staticmethod
+    def push(
+        file: Path = wanna_file_option,
+        profile_name: str = profile_name_option,
+        instance_name: str = instance_name_option("notebook", "push"),
+        mode: PushMode = typer.Option(
+            PushMode.containers,
+            "--mode",
+            "-m",
+            help="Managed-Notebook push mode, due to CI/CD not "
+            "allowing to push to docker registry from "
+            "GCP Agent, we need to split it. "
+            "Notebooks currently support only containers, as we do not create manifests as of now.",
+        ),
+    ) -> None:
+        """
+        Push docker containers. This command also builds the images.
+        """
+        assert mode == PushMode.containers, "Only containers are supported push mode as of now."
```

Comment:
Isn't there a `typer.Exit` or a `typer.Exception`

Reply:
There is typer.Exit we can use